apps/dropbox/vite.config.js

```diff
@@ -4,6 +4,7 @@ export default defineConfig({
   resolve: {
     alias: {
       '@contentful/dam-app-base': './node_modules/@contentful/dam-app-base/lib/index.js',
+      '@contentful/app-sdk': './node_modules/@contentful/app-sdk/dist/index.es.js',
```

Comment:
Is this the addition you think was needed to resolve the problem where it worked on staging but not prod? Curious how you discovered it.

Reply:
I'm hoping 🤞